tools/reviewer/job/sync/GithubReader.java

```diff
@@ -195,53 +195,57 @@ private Thread getCodeThreadByFilenameAndPosition(
                 .filter(comment -> comment.getPosition() == position)
                 .collect(Collectors.toList()));
 
-    Thread.Builder thread = Thread.newBuilder();
+    GithubComment firstComment;
+    if (positionComments.isEmpty()) {
+      throw new RuntimeException("Can not find comments for a position: " + position);
+    } else {
+      firstComment = positionComments.get(0);
```

Comment:
You don't need the else - you can just;
`GithubComment firstComment = positionComments.get(0);`

Reply:
Done.